src/natcap/invest/hra.py

```diff
@@ -214,11 +214,116 @@
             "type": "boolean",
             "required": False,
         }
+    },
+    "outputs": {
+        "outputs": {
+            "type": "directory",
+            "contents": {
+                "TOTAL_RISK_[HABITAT].tif": {
+                    "about": "This raster layer depicts the habitat-specific cumulative risk from all the stressors in a grid cell.",
+                },
+                "TOTAL_RISK_Ecosystem.tif": {
+                    "about": "This raster layer depicts the sum of habitat cumulative risk scores divided by the number of habitats occurring in each cell."
+                },
+                "RECLASS_RISK_[HABITAT].tif": {
+                    "about": "This raster layer depicts the reclassified habitat-specific risk from all the stressors in a grid cell into four categories, where 0 = No Risk, 1 = Low Risk, 2 = Medium Risk, and 3 = High Risk."
+                },
+                "RECLASS_RISK_Ecosystem.tif": {
+                    "about": "This raster layer depicts the reclassified ecosystem risk in each cell."
+                },
+                "SUMMARY_STATISTICS.csv": {
+                    "about": "This CSV file contains mean, minimum, and maximum exposure, consequence, and risk scores for each habitat-stressor pair, as well as habitat-specific scores in each subregion. If the “name” field is not given in the AOI vector, a “Total Region” value will be used to represent the entire AOI extent in the “SUBREGION” column on the table.",
+                    "columns": {
+                        "R_%HIGH": {
+                            "about": "the percentage of high risk areas.",
+                            "type": "percent"
+                        },
+                        "R_%MEDIUM": {
+                            "about": "the percentage of medium risk areas.",
+                            "type": "percent"
+                        },
+                        "R_%LOW": {
+                            "about": "the percentage of low risk areas.",
+                            "type": "percent"
+                        }
+                    }
+                }
+            }
+        },
+        "visualization_outputs": {
+            "type": "directory",
```

Comment:
HRA is fresh in my mind, but I think this should also have a "created_if" flag, pointing at the optional argument of whether to create viz outputs (and maybe that's on the docket for later since this is a draft)

Reply:
ah, good catch!